pygen/pygen_src/riscv_instr_gen_config.py

```diff
@@ -1,34 +1,175 @@
 """
 Copyright 2020 Google LLC
-Copyright 2020 PerfectVIPs Inc.
-
```

Comment:
Add copyright 2020 PerfectVIPs Inc.

Reply:
ok

Reply:
can you please copy and paste this one. There are some missing lines

"""
Copyright 2020 Google LLC
Copyright 2020 PerfectVIPs Inc.
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
Regression script for RISC-V random instruction generator
"""

Reply:
ok I will add these lines.